features/facebookImport/src/static/allStructure.js

```diff
@@ -199,11 +199,11 @@ export default [
     "other_personal_information/no-data.txt",
     "other_personal_information/your_address_books.json",
     "pages",
-    "pages/pages_you've_liked.json",
-    "pages/pages_you've_recommended.json",
-    "pages/pages_you've_unfollowed.json",
-    "pages/pages_you_follow.json",
-    "pages/your_pages.json",
```

Comment:
Interesting one: Shouldn't those stay, because we know they can be in imports (if not the most recent ones)?

Reply:
Right, they should probably stay. Although TBH I'm not sure if this file is still used...